### ISSUE TITLE
请问要让代码在cuda11.0以及pytorch1.7.1的环境中运行应该修改什么内容呢

### ISSUE THREAD

**Issue by tao123322** [Elhoseiny-VisionCAIR-Lab/LTVRR#3]

**Comment by aniket-agarwal1999** [comment]
Hi, can you type your issue in English. I am not able to understand the issue you just posted

**Comment by aniket-agarwal1999** [comment]
Hi, we would have to check it up. I don't think much needs to be changed though. Apart from this we have also provided the environment file which you can use to easily create the virtualenv required for running the codebase.

**Comment by tao123322** [comment]
Our server has minimum support for CUDA11.0, so the environment provided is not good for code to work! I hope you can help me see how to modify. Thank you so much!



---原始邮件---
发件人: "Aniket ***@***.***&gt;
发送时间: 2021年10月10日(周日) 晚上7:01
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [Vision-CAIR/LTVRR] 请问要让代码在cuda11.0以及pytorch1.7.1的环境中运行应该修改什么内容呢 (#3)




 
Hi, we would have to check it up. I don't think much needs to be changed though. Apart from this we have also provided the environment file which you can use to easily create the virtualenv required for running the codebase.
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by YerongLi2** [comment]
Yes, I also got the CUDA version issue,  the error message I am getting is 
`ImportError: /home/wentaoy2/LTVRR/lib/model/roi_pooling/_ext/roi_pooling/_roi_pooling.so: undefined symbol: __cudaRegisterFatBinaryEnd`
